doc/changelog/03-notations/12984-master+import-notation-make-active-again.rst

```diff
@@ -0,0 +1,12 @@
+- **Changed:**
+  Redeclaring a notation reactivates also its printing rule; in
+  particular a second :cmd:`Import` of the same module reactivates the
+  printing rules declared in this module; this leads in theory to
+  changes of behavior in printing though this seems to be mitigated in
+  general by the adoption in `#12986
+  <https://github.com/coq/coq/pull/12986>`_ of a priority given to
```

Comment:
```suggestion
  printing rules declared in this module. In theory, this leads to
  changes of behavior for printing. However, this seems to be mitigated in
  general by the adoption in `#12986
  <https://github.com/coq/coq/pull/12986>`_ of a priority given to
```

Reply:
I used semi-colon so that the parenthesis that comes after is attached to the whole text and not only to last sentence of the text.

Maybe the text in parenthesis can be put after the first sentence though, and the two other sentences after? I don't know well the usages here.

Reply:
I found this one very long sentence hard to read, so I think it should be split up somehow. I have no strong preference how how it should be split up.

Maybe just end a sentence before the parentheses at the end, so that they are more attached to the entire item and less to its last sentence?

Reply:
I prefer to cut the sentence as well. I would also remove the "seems to be".
I mean, either the new behavior designed and empirically tested, or not.
It seems it is to me.

Reply:
@herbelin It's fine to use multiple sentences in a changelog entry. IMHO you shouldn't worry too much about the ending parenthesis.